week-1/K-functions-parameters/exercise4.js

```diff
@@ -1,5 +1,7 @@
 // Declare your function first
-
+function add(a, b) {
+  return (sumab = a + b);
```

Comment:
`return a + b` is sufficient here.

In JavaScript, when you assign a value to a variable without using the `var` keyword, it actually creates a global variable, which are definitely best to avoid! In this line of code you are creating a global variable `sumab`.